tools/compare_ast_symdb.py

```diff
@@ -12,7 +12,7 @@
 CPPCHECK = os.path.expanduser('~/cppcheck/cppcheck')
 
 def run_cppcheck(cppcheck_parameters:str, clang:str):
-    cmd = f'{CPPCHECK} {cppcheck_parameters} {clang} --debug --verbose'
+    cmd = '{} {} {} --debug --verbose'.format(CPPCHECK, cppcheck_parameters, clang)
```

Comment:
I don't think it matters if this devtool is not python 2.x compliant.

Reply:
This is about Python 3.5 compliance - which is still the default on e.g. ubuntu 16.04.

Reply:
ok